### PR TITLE
qat: add support for QAT 402xx

### PR DESCRIPTION
**Pull request by mythi**
Based on
https://lore.kernel.org/linux-crypto/20230303165650.81405-1-damian.muszynski@intel.com/

### CODE REVIEW

**Review by hj-johannes-lee — DISMISSED** [review]
LGTM!

**Review by hj-johannes-lee — COMMENTED** [review]
Btw, what about [pkg/contollers/qat/controller.go line #210](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/pkg/controllers/qat/controller.go#L210)?

**Comment by mythi** [comment]
> Btw, what about [pkg/contollers/qat/controller.go line #210](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/pkg/controllers/qat/controller.go#L210)?

thanks, of course!

File: demo/qat-init.sh [hunk 1]
```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 # This script is based on qatlib's qat_init.sh
 NODE_NAME="${NODE_NAME:-}"
-ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-37c8 4940 4942}
+ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-37c8 4940 4942 4944}
```
Comment (tkatila) [inline demo/qat-init.sh:4]: Out of curiosity, why is the indexing different (-1) here vs. the go code?

**Review by tkatila — DISMISSED** [review]

File: demo/qat-init.sh (hunk 1 above)
Comment (hj-johannes-lee) [reply demo/qat-init.sh:4]: Maybe..
here: PF
dpdkdrv: VF ?

Comment (mythi) [reply demo/qat-init.sh:4]: @hj-johannes-lee probably remembers

**Review by hj-johannes-lee — COMMENTED** [review]
One more small thing found out:
Readme line 46, 137

**Review by hj-johannes-lee — APPROVED** [review]

**Pull request merged**